README.md

```diff
@@ -345,7 +345,7 @@ To define a test, use `defineTest` from the `testUtils` module:
 
 ```js
 jest.autoMockOff();
-const defineTest = require('jscodeshift/src/testUtils').defineTest;
+const defineTest = require('jscodeshift/dist/testUtils').defineTest;
```

Comment:
Just a precaution.  I don't want anybody relying on `src/` having a certain structure or contain certain files. `dist/` is the folder that is "exposed".